lib/abstract-generator.js

```diff
@@ -80,6 +85,10 @@ class AbstractGenerator {
         rdfaVocs[prefix] = yarrrml.prefixes[prefix];
       });
     }
+
+    if (this.authors.length > 0) {
+      this.generateDatasetDescriptionWithAuthors(this.authors);
```

Comment:
Is there a specific reason that dataset metadata is only generated when authors are present? I'd propose always adding the dataset metadata (one more reason to use YARRRML), and updating the spec accordingly (`a YARRRML Parser MAY add dataset metadata, but MUST add dataset metadata when authors are described`

Reply:
No specific reason. Will change to code so that always this metadata is added :smile_cat:

Reply:
Updated :rocket: